### ISSUE TITLE
Failure compiling with webpack

### ISSUE THREAD

**Issue by leviskim17** [ognjen-petrovic/js-dxf#92]
I am trying to build the dxf-writer with webpack. 

But it is failed with the below two part. 

**in Handle.js**

`
class Handle 
{
    static seed = 0; <<<<== ERROR
    static next() {
        return (++Handle.seed).toString(16).toUpperCase();
    }
`

**in TextStyle.js**
`
class TextStyle extends DatabaseObject 
{
    fontFileName = 'txt';  <<<<== ERROR
    constructor(name) {
        super(["AcDbSymbolTableRecord", "AcDbTextStyleTableRecord"]);
        this.name = name;
    }
`
    
When I change the parts as like below

**in Handle.js**
`
class Handle 
{
     /**
     * @param {number} seed - Center x
     */
    static next() {
        return (++Handle.seed).toString(16).toUpperCase();
    }
`

**in TextStyle.js**
`
class TextStyle extends DatabaseObject {
     /**
     * @param {string} fontFileName - End angle
     */
    constructor(name) {
        super(["AcDbSymbolTableRecord", "AcDbTextStyleTableRecord"]);
        this.name = name;
    }
`
    
I check the compile is succeeded and works in temporal. But I am not sure whether it is working logically correct on your intention or not.

Can you confirm about this approach? And if possible, please update the part for webpack building. 
(I have tested wepback v4.42.1 and the latest)

**Comment by ognjen-petrovic** [comment]
Hi,
That is Javascript,  how do you complile?

**Comment by tarikjabiri** [comment]
Hi

I think he means bundling because webpack is a bundler so there is some new syntax used there so I think the latest webpack support that syntax but the old version I think it needs a babel loader I am not sure. Personally I use rollup for bundling or vite for web apps.

Or we need to bundle the code using rollup so it will generate supported esm and cjs js code.

I keep talking about rollup because I just familiar with it.

Regards

**Comment by leviskim17** [comment]
Hi @ognjen-petrovic,

Let me share my experience more. I am implementing a POC to combine webgl three.js & your dxf-writer and I have met an coding issue related to esm and cjs.

For solving the coding issue, I have tried to several ways, but the result was not good. So I decided to try babel loader. however, I met "no loader issue" or "no parser issue" with the part that I point-out upper in the final. I have tested from babel-loader v8 to the latest. 

Because I am not a javascript expert, I am not sure about my view though. I think the fundamental issue seems to come from that webpack cannot understand the part. Even though I adopt the babel-loader to help the issue, babel-loader seem cannot help it too.
